src/allmydata/web/status.py

```diff
@@ -489,50 +366,85 @@ def render(self, request):
         return json.dumps(data, indent=1) + "\n"
 
 
-class DownloadStatusPage(DownloadResultsRendererMixin, rend.Page):
-    docFactory = getxmlfile("download-status.xhtml")
+class DownloadStatusPage(MultiFormatResource):
+    """Renders /status/down-%d."""
```

Comment:
I think maybe just `Resource`? (`MultiFormatResource` is for pages which can be returned in HTML and JSON).

Reply:
Made `DownloadStatusPage` a `Resource` in b14c0a0, and also an `object` in 00e852c.  Consequently `render_HTML()` in this class is now `render_GET()`.